File: src/vb/Accounts/Account.vb
Namespace Accounts

Public Class Account

    Public Property AccountNumber As String
    Public Property Balance As Decimal

    Public Sub New(accountNumber As String, initialBalance As Decimal)
        Me.AccountNumber = accountNumber
        Me.Balance = initialBalance
    End Sub

    Public Sub Credit(amount As Decimal)
        Me.Balance += amount
    End Sub

    Public Sub Debit(amount As Decimal)
        Me.Balance -= amount
    End Sub

End Class

End Namespace

File: cs_generated_v1/Accounts/Account.cs
namespace Accounts
{
    public class Account
    {
        public string AccountNumber { get; set; }
        public decimal Balance { get; set; }

        public Account(string accountNumber, decimal initialBalance)
        {
            AccountNumber = accountNumber;
            Balance = initialBalance;
        }

        public void Credit(decimal amount)
        {
            Balance += amount;
        }

        public void Debit(decimal amount)
        {
            Balance -= amount;
        }
    }
}